github/Organization.py

```diff
@@ -577,9 +582,18 @@ def create_repo(
             post_parameters["allow_rebase_merge"] = allow_rebase_merge
         if delete_branch_on_merge is not github.GithubObject.NotSet:
             post_parameters["delete_branch_on_merge"] = delete_branch_on_merge
+        if visibility is not github.GithubObject.NotSet:
+            post_parameters["visibility"] = visibility
+
         headers, data = self._requester.requestJsonAndCheck(
-            "POST", self.url + "/repos", input=post_parameters
+            "POST",
+            self.url + "/repos",
+            input=post_parameters,
+            headers={"Accept": Consts.repositoryVisibilityPreview}
```

Comment:
And then down here, you just have ``headers=post_headers,``

And also, no fair apologising for learning! Python is fun, and I'm always delighted to help people learn one of my favourite languages.

Reply:
Makes sense. Thanks